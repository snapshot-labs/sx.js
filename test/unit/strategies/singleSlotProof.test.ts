import singleSlotProofStrategy from '../../../src/strategies/singleSlotProof';
import { envelope, metadata } from '../fixtures';

const ethUrl = process.env.GOERLI_NODE_URL as string;

describe('singleSlotProofStrategy', () => {
  it('should return type', () => {
    expect(singleSlotProofStrategy.type).toBe('singleSlotProof');
  });

  it('should return params', async () => {
    const params = await singleSlotProofStrategy.getParams(envelope, metadata, { ethUrl });

    expect(params).toMatchSnapshot();
  });

  it('should return extra propose calls', async () => {
    const params = await singleSlotProofStrategy.getExtraProposeCalls(envelope, metadata, {
      ethUrl
    });

    expect(params).toMatchSnapshot();
  });
});
