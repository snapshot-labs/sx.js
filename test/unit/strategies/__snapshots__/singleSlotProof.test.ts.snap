// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`singleSlotProofStrategy should return extra propose calls 1`] = `
Array [
  Object {
    "calldata": Array [
      15,
      7541970,
      "0xb4fbf271143f4fbf",
      "0x7b91a5ded31805e4",
      "0x2b2208d6",
      7,
      "0x214",
      "0x214",
      "0x214",
      "0x214",
      "0x214",
      "0x174",
      "0x73",
      7,
      "0x43",
      "0x43",
      "0x43",
      "0x43",
      "0x43",
      "0x2f",
      "0xf",
      397,
      "0xf90211a090c45fa7",
      "0x4fc4a29c0aac342e",
      "0xaad55a0e99515ddd",
      "0x1f3c3f3a21bf3750",
      "0xb83a57cda0a67e36",
      "0xe707d99ef9ef30d9",
      "0xccd970384adbc9a5",
      "0xb0d1c289549ff3fa",
      "0x5d504b1beda0fd66",
      "0x2bc457f4e425dbce",
      "0x56d4b58f18f63b9d",
      "0xbf5b46af33cbd7e0",
      "0x415cc432ed1da0bd",
      "0xf0b21ac8854ea548",
      "0x1ff83d71b568b90f",
      "0x2f9449fce78a98c3",
      "0xb1544bc92b3613a0",
      "0xcd93bc1bc3c017db",
      "0x35500d469dc0ab9d",
      "0x79ef984629db9d10",
      "0x12cc792dee5a672b",
      "0xa035eb23e41807ba",
      "0x63bab777884dc840",
      "0xbd7750cfc50968b6",
      "0x6fcd1c545437eade",
      "0xada0ef702473aa22",
      "0xd553f90cd2b9b7e9",
      "0x490ef603a7208562",
      "0x84fe320c1427b4d7",
      "0x4023a0116bed9fcf",
      "0x219434ade705e30d",
      "0x91c9039f5be821b3",
      "0x0f4d456e53e66799",
      "0xdccf97a04f9c54d6",
      "0xd33856760d20e8fb",
      "0x343bd609842c4305",
      "0x4426b8358d2c0f6d",
      "0x060da280a0566277",
      "0x67433dc31cbcd2e6",
      "0x24ae1aee65566b6e",
      "0x7ee837b051294523",
      "0xcd809b5831a05b42",
      "0x0bd2baf3b7e823f6",
      "0xf6e98977bff4797e",
      "0x38076d932dc03534",
      "0xf5564cd774cfa0d0",
      "0x612d6aa6f4f3c0c2",
      "0xa0e8758ece802671",
      "0x4405d0eb9896fa83",
      "0xf86cf033df0c40a0",
      "0xcdaeba31eabb42ef",
      "0xc7851e82fa57191d",
      "0xfac2fa03a191b2c3",
      "0x1e7278ebcb47a708",
      "0xa0c056d96cb1a9c6",
      "0x28fe7b71f575c3a7",
      "0x928d003b9b00ee6b",
      "0x71e7828f6b7d1772",
      "0x9ba0ba9719a5c3b4",
      "0x4712685c62561b9d",
      "0x31e6f622166ffd12",
      "0x472480dfe69044ae",
      "0xd881a0bfa932618f",
      "0x36a7f0e8ceedcfee",
      "0x69a8036d3ad837cf",
      "0x4f4193bb2880d632",
      "0x5a41d280",
      "0xf90211a0af716862",
      "0x3781369a26a383c3",
      "0x261e918e7829f1fb",
      "0x98e06148dd169676",
      "0x097fbff6a0c7e10d",
      "0x2c7ea11566e0bfe4",
      "0xc87889bc7b1f7340",
      "0xb70468643bb16026",
      "0x5efe717714a08eae",
      "0xd98c9250aa38406e",
      "0xa51817e9f81d3f6c",
      "0x8bafaef0c86a40ef",
      "0x5dfd3bbce2dba045",
      "0xd8573e8e10ed7230",
      "0x35cd974909c467f4",
      "0x3e6f29212f6d74cc",
      "0xf354713c8c726aa0",
      "0xfc3013c6bd02f904",
      "0x02bcb3610ec1dab5",
      "0x51cd500ecfb6c2d7",
      "0x8172604589989b02",
      "0xa0543a75b4d278d9",
      "0xf637ab289d11b2ad",
      "0x370e0d709ea4e208",
      "0x96002598fd73d51f",
      "0x62a0aa8def852a2b",
      "0x59eae0d17fbf169d",
      "0x69f916a7df16a3d9",
      "0xa0d5718d7bb42565",
      "0xe9c0a0150289c072",
      "0x0999408a2cff424c",
      "0xc41636f4dda66b8a",
      "0xbdfbc535ab3ca49e",
      "0xc85540a078978028",
      "0x38a5968bd0544c11",
      "0x513f9abd2341675a",
      "0x3fe299d2a65353fc",
      "0xc610d5dfa00f78c7",
      "0x15881b20a63488b6",
      "0x3671f08fbd4322fb",
      "0xd7895b43cef9ce25",
      "0x7705498beaa06ae3",
      "0x47f1f47832b9bd2d",
      "0x39951a9d6ff7fd73",
      "0xe99185eecceb2a59",
      "0x4a534834f135a075",
      "0xf8c08a308f2177d0",
      "0x96ec447295cbc881",
      "0xf8c733cd1c911602",
      "0x90a29053b458bfa0",
      "0x41e5059db8b41af8",
      "0xd4af9b76dacc38d3",
      "0x916ed9908cf5e615",
      "0x30f759baa74bfb4f",
      "0xa0d4fc9ab020524c",
      "0xbe28743447a81b21",
      "0x9fea21b97601604c",
      "0xe3a77461361f3351",
      "0x38a02fcedc6fc350",
      "0xc81a265f2f3da4f6",
      "0xec0640475706cb55",
      "0x5c0e4e999690c6f5",
      "0xc76da0c8ae027a1f",
      "0xed4de4a9572fe133",
      "0x1edb41f3c947188b",
      "0x145a9b51cc7b3456",
      "0x7a1b2b80",
      "0xf90211a0621ac2fd",
      "0x5c2d8b1998437321",
      "0x9d48faa3f7410c16",
      "0x62ca42244435fb42",
      "0xc30ae194a0e1a740",
      "0xc6f49c74b224ec75",
      "0x118f0b29b9b1a178",
      "0xf224864596fba8a3",
      "0xcf5cbb00eca0956d",
      "0xc450fd34a7028674",
      "0x7104292ddbff7515",
      "0x20724ae0eaaf0ebb",
      "0x2a3c7d1814baa0ce",
      "0xb08aab248e36ad7a",
      "0x93a1e9d11e0d54ce",
      "0x2e279df413a7c20f",
      "0xa878c294246330a0",
      "0xfb529680cf54c7cf",
      "0x23b508963171653f",
      "0x2e89bc64bb75b622",
      "0x9c08f8cafd8ff038",
      "0xa07c7dac740a9317",
      "0x71d274533e4f1da6",
      "0x94a4f0c977492940",
      "0x95d6b5231286466c",
      "0x2fa02dd57a0a39e2",
      "0xe19e36833cf4968f",
      "0x3d99069113b316db",
      "0x1bca067f84c20579",
      "0x61a6a0ab496beb6d",
      "0xc38924619c965cda",
      "0xc2a49cfb348d71d3",
      "0xb1c4293e629b4304",
      "0xd95425a03b6d7985",
      "0x8a9d71e627a560f4",
      "0xb65bed9b58f73d55",
      "0x05b25905d604ea5f",
      "0x1010aec6a07aff04",
      "0x49137517992b4f54",
      "0x5e5b7a21bbcbedec",
      "0xcfa7222e6f4e3527",
      "0xbbb9aa1552a0e95a",
      "0x5492bb2bfcd1c0d2",
      "0x6465da4d2c1c80f9",
      "0x26d98ad266cb66c2",
      "0xd4ae51961b2fa058",
      "0x4842743a860a90a9",
      "0x9b63957ddb6469bf",
      "0x5bdd1b7b349ef031",
      "0x7c252a6cfb705ca0",
      "0x29b1a0f2b9c63687",
      "0xaf522a5add7b1b65",
      "0x4460abd81d77ed02",
      "0x75f3975d5f7927a2",
      "0xa0c203cbecb017c5",
      "0x10083e6ca75e3ed7",
      "0x9fd65315f8d8fb04",
      "0x7302c4856a15b34c",
      "0xb2a0685829813d1f",
      "0x69068b8cc72d79e8",
      "0x7c6878ea0252584d",
      "0x06c475e075fa34a5",
      "0x4743a04c906c4158",
      "0x78cdae07885e7b1c",
      "0xcab00e8555490f8d",
      "0xf8c3a7a8a87748be",
      "0x2e0aeb80",
      "0xf90211a07e7ab7e2",
      "0x7b1a5893668ae371",
      "0xa797e1f4295d68c7",
      "0x143ab4755153ee1e",
      "0xb2a435a9a0b5a5d6",
      "0x38aa1f452b843395",
      "0xce4c12b48b9e543e",
      "0x13024b0f36d1fb66",
      "0xabfc3f5e84a07f75",
      "0x94cf69d0cb1f36b4",
      "0x4b9f0988045aa9e3",
      "0x43a06c00813112cc",
      "0xc091bca0610ba0b9",
      "0x935cd611f6da7a2c",
      "0x64483dd9d18a922c",
      "0xd59b89b04902e384",
      "0xe9e27be7c8be9fa0",
      "0x4229cff58a91dbaa",
      "0xa2e068ba0c0c4a83",
      "0x7875cae2f8363f24",
      "0x531da2f015c98195",
      "0xa04ccec097b66167",
      "0xf3951fe3666bf7c1",
      "0xc5a37cc7b5b1262e",
      "0x070036aade183f30",
      "0xdda0f92e23af9f43",
      "0xf5a74d87489e7380",
      "0xe540374909582e86",
      "0xc7b6563405151261",
      "0x3667a077abc41fcf",
      "0xea5f03c7b3821478",
      "0x0a7b71235d9598aa",
      "0x6c5e54975e77cba7",
      "0x0fb821a022cd7631",
      "0xd128f700f3890489",
      "0x20de72ea3b42f801",
      "0xe014bd2ad134f65d",
      "0x45db77daa0f0e773",
      "0x01cc75a9c4c44fec",
      "0x4aee1bc706cbcf1c",
      "0xc3a1228e53b301bf",
      "0xbb65fde090a067a3",
      "0xc21f423442d602f3",
      "0x80548f5d4502e9a4",
      "0xe5b274cd182b6d19",
      "0x742af82f26b7a078",
      "0x068a01e08448f65a",
      "0xfccdd18aba6030da",
      "0x46f806df6fb740c3",
      "0x78f3b9f348ed7ba0",
      "0xf9fd6dbce7ad091d",
      "0x987bfab0198863e8",
      "0x1a02cf3595b8034d",
      "0xfeeffcdcfda903ab",
      "0xa0e6b49ef4c0f200",
      "0x48ece865a0c8e604",
      "0xa5e1ab8de64258ab",
      "0xcf4b653370e3e8fb",
      "0x20a0afbe6b669e53",
      "0x714119b3e54bd7f4",
      "0xfbef502398b8bb71",
      "0x6743cb28e4f98136",
      "0xe27ba0835efef912",
      "0x8c92f54182cc1c2e",
      "0x6e2de760d1a276cf",
      "0x7dc3191ea86f2fd2",
      "0x4d5eac80",
      "0xf90211a09ebd199e",
      "0x9610b8f150e6a1ba",
      "0xe35878e8dded69a4",
      "0x41e4626befd927d4",
      "0x331b41a5a0e5f906",
      "0xbb34d649e44689bf",
      "0xf38e82ba2c87cb9c",
      "0xc81965afe8805725",
      "0x1adcd0a52fa00c57",
      "0x9884526e798e5120",
      "0xf3d66329b3cc8300",
      "0xa74eaffdbc9d0abc",
      "0x43c559712ef8a0cf",
      "0x99c0e31a0696a111",
      "0x25d7dbb8f18c569c",
      "0xc25cea2364748246",
      "0x0afcf9ed3311cca0",
      "0xd2798640e4354aee",
      "0x7ec677fe9a23318a",
      "0x67e8f60c1a57e867",
      "0x25f4bb122db412c1",
      "0xa019e79945ce5d3f",
      "0xc5bbd5e95497d75d",
      "0xc22f59801f1e7cb8",
      "0x299bf81a68cd7581",
      "0xd3a052497ea2f0a1",
      "0x0a26b88ab09d69fa",
      "0xb651dd1844ec1310",
      "0x6e8511583fcbe0b0",
      "0x8f40a0d36a3e6991",
      "0xf7ca38ee895b478f",
      "0x75e500973a9d165e",
      "0x9b995509f3a637f6",
      "0x8a08d2a0e387e969",
      "0x8d9fa5b518161d79",
      "0x3ca477a47118f4f1",
      "0xc0ff25bc4fb388e9",
      "0xf1e36557a0f502e9",
      "0x811218090a5bfb2e",
      "0x0aeb94d7e9748030",
      "0x77c544d990da48d8",
      "0x06bf96e770a04be0",
      "0xd6907420a7823823",
      "0x1cbad51b962b4683",
      "0x06b1f5bc03dc682e",
      "0x4dae29acbd74a0c3",
      "0x4d5e5b40a7ce2acb",
      "0x8785d9c49520678b",
      "0xf198dedb07dffcf1",
      "0xadee44af1d8468a0",
      "0x45e86c5b5ea86598",
      "0x56e408bb11f56e9e",
      "0xfbbdc266ec21cd6c",
      "0xd78870447eecb5a0",
      "0xa09724ffd336cf07",
      "0xdf9a9a3ac9feb7b4",
      "0x1d1b1379438a0664",
      "0xceb5afef58173f9e",
      "0x32a063f811b272d8",
      "0x46d6119ba3bf0051",
      "0xdebd7db00d1e928a",
      "0x1684fade501229e4",
      "0xd4a0a06ec48582d9",
      "0x853cc28276bb73fa",
      "0xaa9ece122671d9db",
      "0x4fb1982e76ac1e56",
      "0x2f04e680",
      "0xf90171a02fcb5ab3",
      "0xfdea4aecd8121a49",
      "0xfe16f13dab9cc242",
      "0x04779d29a292dbc0",
      "0xb2c75228a0d4448a",
      "0x82f6c9128162ae15",
      "0x08d1d373c4fde725",
      "0x2859f247d2bccc0d",
      "0xdaff99fe8da0322b",
      "0xe735938a0d7e5c6f",
      "0x70246dc6f5a9231a",
      "0xc252461e8788c97f",
      "0x80b311fc59a080a0",
      "0x53ac8d8823a1abdd",
      "0xb1bc30d2330b11ed",
      "0x4e82d7f597cd9215",
      "0xf930a7ead762b04d",
      "0x80a0202c8b9187a2",
      "0x228987cf895d8e0b",
      "0x02b4584f256cbca0",
      "0x43e18db0c56564db",
      "0x5c128080a0034e99",
      "0xbb970d38fd7dd1aa",
      "0xca51bb7ed3bddd2a",
      "0xdd0b1b8eacc86990",
      "0xa80e0071b6a08586",
      "0xada2ef82ba5393d9",
      "0xbc874f4975296380",
      "0x59b62af37f5f2153",
      "0xeaa9980aaf6580a0",
      "0x6f335f8cf3c1bc9c",
      "0x3fae16fefa39aa8b",
      "0xaeb1bc537a81a674",
      "0x2e906f4bbb095889",
      "0xa0bbd2b83030d4d4",
      "0xe6a8f8bf17b44252",
      "0x12abde375f20e1f5",
      "0x4c7b36bd167f928b",
      "0x81a0f65fad8c91c1",
      "0x1e62922625a9e06c",
      "0xe6afc7bb6c9110c4",
      "0x008fe46d88dcc1cf",
      "0x5bdfa0bc2641ead2",
      "0x2cdff68ac20533e5",
      "0xbbd3b412b1d1a48c",
      "0x2be3a09627e10497",
      "0x3b638f80",
      "0xf8719e20294dda2d",
      "0x5979f269acd2630d",
      "0xff3880bca6301564",
      "0xc80e5af071862b4e",
      "0x26b850f84e018a45",
      "0x979cb22102a9fdc7",
      "0x43a07594ddd72120",
      "0x464fc5a1161288b5",
      "0xd321004aa3c17d89",
      "0x58654db3516ac408",
      "0x4a4ba08b3eb75e29",
      "0xf2a9514d1a29784d",
      "0x34f29fd79929d886",
      "0x2345b0c70e467d57",
      "0x6b805d",
    ],
    "contractAddress": "0x363108ac1521a47b4f7d82f8ba868199bc1535216bbedfc1b071ae93cc406fd",
    "entrypoint": "prove_account",
  },
]
`;

exports[`singleSlotProofStrategy should return params 1`] = `
Array [
  "0xf76ce3c72ae592d6",
  "0xe400df3a685e12d2",
  "0x1477cde4d6e44a47",
  "0x7fa1a8f7bd1b8dc5",
  "0x5",
  "0x214",
  "0x214",
  "0x214",
  "0x194",
  "0x2a",
  "0x43",
  "0x43",
  "0x43",
  "0x33",
  "0x6",
  "0xf90211a059bc304b",
  "0xa37e650a793b4424",
  "0x1a17a5b9233f2795",
  "0x6b493d6c340ea59d",
  "0x13a6da60a0186a4e",
  "0x92c4ad3aa0c2a24f",
  "0xa928293465cd2960",
  "0xa00c93910c404991",
  "0xf71fe2d991a05730",
  "0xe64338c20057963f",
  "0x0e64559136dc51ca",
  "0x14c22297f2c35412",
  "0x4e58f127ee9aa0a8",
  "0x352eba1e82d8b4d0",
  "0x1788921effdd7d0f",
  "0x87604bf0e83357f5",
  "0x35bebee3436d49a0",
  "0xdf47d08791ec3428",
  "0x549c51baf3961327",
  "0xb597a6109ba2a701",
  "0x0592fa52860371bd",
  "0xa011776001b4c95f",
  "0xf61e64c193f3678d",
  "0xdf11b67bfecee4b0",
  "0x5da1939a7cb25054",
  "0x63a07f467be2a58b",
  "0xa3dfc636110cb129",
  "0xcee6ebb12aba1b03",
  "0xfcaf122d3207b3bd",
  "0xb9e3a0e0cb68c120",
  "0xb722250050ed281a",
  "0x658ab17861802a14",
  "0xf184ed37dbb584c1",
  "0x697712a0a1c5390f",
  "0xe5a0f79763f8142e",
  "0xf735e1f53898a3f2",
  "0xc3fb9e107ffeb653",
  "0x0c42862ca07af116",
  "0xcc4790fb552d36aa",
  "0x2a7b30b3039a6467",
  "0x38bd44205af6d320",
  "0x9914c58e42a09d66",
  "0x2822f57a28542311",
  "0xbf87dc8039ac0f66",
  "0x9d7bed7319a36893",
  "0x5af0acf681fea03e",
  "0x45d26ee8bd44fc0c",
  "0x6b56ae21c04bafd2",
  "0xb42744a05e609757",
  "0xb8c12395497b32a0",
  "0x3e9d7067350bf827",
  "0xc3b86a7cf4fe2d15",
  "0xae9de4eba427a0ba",
  "0x8b27190fcd485cf2",
  "0xa0b8533340ffa555",
  "0x26f4ab96fec43236",
  "0x8abf6b2c40cc1348",
  "0x3ee79da5b7ab7e6b",
  "0x74a0b0fa41ddea2c",
  "0x66811a1edd08c060",
  "0x8e2129ee90cdc18e",
  "0xfb330828b13676bf",
  "0x824aa0643ab93773",
  "0x86bcca78bf4b747d",
  "0xf5717631e9565377",
  "0x6cb25a020a4e20c5",
  "0x2ed39280",
  "0xf90211a09fe437f6",
  "0x5de70a069dcea8e6",
  "0x86fd522ec322f958",
  "0x9a0fd1a35b91212b",
  "0x1882caf1a05f7b4d",
  "0x61d3086773dd4e58",
  "0xa9a0f1e115b1a289",
  "0x928c33221cbedc6a",
  "0x976c4e2841a04619",
  "0x892b9271bd79e572",
  "0xbc5dd4336e7a815c",
  "0xfde5e27c4d7bcd5c",
  "0x03b830755ea5a043",
  "0x6393b8c762c61fe5",
  "0x5aeae3c498c02604",
  "0xb49487de372408c3",
  "0x28ac828abda0a9a0",
  "0xcecad3e685ef2282",
  "0xfcd88bd4eb1556f0",
  "0x350f254829789c2c",
  "0x8e59fc194e02c284",
  "0xa05d192478be6ff7",
  "0xebfad7f9c82a1d8f",
  "0x2386800af821fdda",
  "0x739d62d45d688584",
  "0x7aa0c334fa98ba3e",
  "0xaaeb23825abb90fc",
  "0x48bc315cbfa1cf90",
  "0x9dd065e8661d058b",
  "0x12d5a00e35058f67",
  "0xa6f782e83b64eaa3",
  "0x8569bba27fc0db09",
  "0xa9dfe5256b59165f",
  "0x6f975ca0ea9f7c72",
  "0x05bd6976b695dc87",
  "0x6c39d1ab960d7530",
  "0x81a1200a9bec7064",
  "0x938baec3a0efa702",
  "0x40ec52590351c647",
  "0x748d91fd0737ab7d",
  "0x997a9915a68aac55",
  "0x2d61efb5fba098ee",
  "0x26b3a64168669fa1",
  "0x5863d5a8830623f4",
  "0x8e6e46a06eb39bd7",
  "0x101ddf2aa149a031",
  "0xf76b3c8f115d63dd",
  "0x4f03b59d02ed3933",
  "0x300c93ff543b7e55",
  "0x9fa07f87dc1f1ea0",
  "0xf417ff37993f89a2",
  "0xa6d28480630e6994",
  "0x3192cdacc08867e7",
  "0x1dfccc1b3cb7e297",
  "0xa0f21293cd92f825",
  "0x4a1bc00ee9b214d7",
  "0x383bf740048f12f1",
  "0x8b774aa5e76ca193",
  "0x83a023a8b3b7a6fd",
  "0xad4827a36b866d9a",
  "0xba75a2c682aed165",
  "0x0d8c65781973ae20",
  "0x98eca04be726d0b2",
  "0xc990eb2c278a49dc",
  "0x0b036b5f191bbe3e",
  "0x855e6a3d748ec867",
  "0x4ce63380",
  "0xf90211a017b1e13c",
  "0x6508ccae5e66dea0",
  "0xed0b0d783ea86d1c",
  "0x2a324e2173c6ee36",
  "0xdbddffc4a014da6c",
  "0x9e234dc7d20d0882",
  "0xb8ee89952ae4f043",
  "0x80cfdb1b2c444626",
  "0x78dec05254a051ce",
  "0xe7c854cefd55644f",
  "0xc2b0a1303c816367",
  "0xfdd7a6a231c6ac19",
  "0x8262dc497e2da0d6",
  "0xc71444003c1f0bb5",
  "0xcd330b096667ac63",
  "0x72e22aadf5b5ae1c",
  "0x63ee2b58ca009da0",
  "0xc0e0ddc3b942bb48",
  "0xd75858cb3c11908c",
  "0xeca01ea555127162",
  "0x9590323cf593ced3",
  "0xa0ed46597f9b9b32",
  "0x48669ef3a5fabfef",
  "0x35f716209f946655",
  "0xfc8f8b37b2d5671d",
  "0x4da0cd6661d01f7e",
  "0xf28f037ad0fad4fb",
  "0x28238977de6d957f",
  "0x399ef84629a560dc",
  "0xe141a0e17d6a2213",
  "0x51f0734f5e54737c",
  "0xa36a23d105e96c45",
  "0x022126429a0dc392",
  "0x153d68a0333da69d",
  "0xa07d8cab0fa960d2",
  "0xf272515c7b4a49f1",
  "0xcbfd48c8b7ba05b2",
  "0x5a33e785a0812ab6",
  "0x027efed15864af9e",
  "0x4f54d42fc851e142",
  "0xa7c6bf167ef6862b",
  "0xb852c71ffea09e39",
  "0x01135f677564069c",
  "0x765ee715b6e5545f",
  "0x10673e66eeb7b3ce",
  "0xf59301463a48a0f9",
  "0x3a97198b35a6e54a",
  "0xbb5511c3e51d50f0",
  "0x000d1dfc1f1e0114",
  "0xd8393ad6618bbea0",
  "0x53e828c905c037bc",
  "0x3fb9e6406df804e5",
  "0x37d7747dd52da3a1",
  "0x0e4d7cc8027f20cf",
  "0xa0ef49bff95772c2",
  "0x58e648c5b9a982a1",
  "0x71c2885e35f6068e",
  "0x32f9bf3e83f6f724",
  "0xd1a0d7191ae69b27",
  "0x99813a6c9d441b03",
  "0xb3a161156f769f31",
  "0xb1e68c6cf7ce1dc3",
  "0x4a30a06eeb0243e1",
  "0xeccb007457b1eab2",
  "0x65f9a95177795a7c",
  "0xf083fe7fbc69fc5e",
  "0xcab19180",
  "0xf90191a01007767b",
  "0xb344c4e87e37cc90",
  "0xf8a00da7ed054250",
  "0x4d606c28226f27e2",
  "0x4c82dcdca02ac8c7",
  "0x2d4658a504f42428",
  "0x03a66f738b467f60",
  "0xd9f8ba8f6fbb43c1",
  "0x5c6a20edfaa06faa",
  "0x897c9c1bbdec08a4",
  "0x19904e8ffae90af5",
  "0xf1eff29c7f8906f7",
  "0x70a5694acf3f80a0",
  "0xacfc68e6f2ee94ae",
  "0x4fc6212683b82b65",
  "0x987bc7c5f0652bac",
  "0x8b5e38e813e98863",
  "0x80a07e66e4387a3a",
  "0x7d4087264fccc3b5",
  "0xbbc21931cf258085",
  "0x190a85a5e13387dc",
  "0x580d80a090b63bb3",
  "0x32d5c0ada40c062b",
  "0xd2c7f107f0be01bb",
  "0xb369402388d58b15",
  "0xc2d807f7a0ac88c0",
  "0xff5fa3a06f7d6806",
  "0xf9fd9d2299fd6ee3",
  "0xd624ea28a9083d05",
  "0x76616b0acfa061cd",
  "0xf1584c7f31be758f",
  "0xd416ea7324889b7c",
  "0xff7ea45ed2bf8af2",
  "0x37055cbe5d9ea094",
  "0x6d3161d428275b25",
  "0xcb8d7727cb86afc3",
  "0xce00f988063455b6",
  "0xd1f1f7f7155f2e80",
  "0xa07c8d594cfe94f9",
  "0x996fe9874cf27458",
  "0x923454cb7b9a2c22",
  "0xd05e088a591bc1f7",
  "0x51a0e175414c25c2",
  "0x2f0795b2b2ee5a4f",
  "0x6808684c421cb04d",
  "0x47b53ba4c9a9b2cd",
  "0xffb8a0497d3d0802",
  "0xd2671a904b72f074",
  "0x1e3121f1b059da34",
  "0x1d72ec17dd5461b3",
  "0x252ca080",
  "0xe99f202b8fa733b0",
  "0xefb796ea597c3c43",
  "0xb9f4433c5b8b1e68",
  "0xbee85cd88cd3e5a0",
  "0xbf88873ff2e795f5",
  "0x0000",
]
`;
